<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Evan Teran</author>
 <class>BinaryStringWidget</class>
 <widget class="QWidget" name="BinaryStringWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BinaryString</string>
  </property>
  <layout class="QGridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="txtHex">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtAscii">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblUTF16">
     <property name="text">
      <string>UTF-16</string>
     </property>
     <property name="buddy">
      <cstring>txtUTF16</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblAscii">
     <property name="text">
      <string>ASCII</string>
     </property>
     <property name="buddy">
      <cstring>txtAscii</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtUTF16">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblHex">
     <property name="text">
      <string>Hex</string>
     </property>
     <property name="buddy">
      <cstring>txtHex</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtAscii</tabstop>
  <tabstop>txtUTF16</tabstop>
  <tabstop>txtHex</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
